config/initializers/sentry.rb

```diff
@@ -1,9 +1,11 @@
-Sentry.init do |config|
-  config.dsn = ENV["SENTRY_DSN"]
-  config.breadcrumbs_logger = %i[active_support_logger http_logger]
-  config.debug = true
-  config.traces_sample_rate = 0.0
-  config.environment = ENV["PAAS_ENVIRONMENT"] || ENV["SENTRY_ENVIRONMENT"] || "local"
+if Settings.sentry.dsn.present?
+  Sentry.init do |config|
+    config.dsn = Settings.sentry.dsn
+    config.breadcrumbs_logger = %i[active_support_logger http_logger]
+    config.debug = true
+    config.traces_sample_rate = 0.0
+    config.environment = Settings.sentry.environment || "local"
```

Comment:
Just curious, is the `|| local` necessary since Settings.yml seemingly defaults it to `local` on line 20?

Reply:
good spot, I'll update this one and small pr for forms-api too so they match